Modules/Venice/Sources/Venice/Poll/Poll.swift

```diff
@@ -17,7 +17,7 @@ public struct PollEvent : OptionSet {
 }
 
 /// Polls file descriptor for events
-public func poll(_ fileDescriptor: FileDescriptor, events: PollEvent, deadline: Double = .never) throws -> PollEvent {
+public func poll(_ fileDescriptor: FileDescriptor, events: PollEvent, deadline: Double) throws -> PollEvent {
```

Comment:
Because people should take a conscious decision about the deadline. Maybe some places we could add the default, but I don't know. The reason sometimes the tests were hanging is that some operation fails but never returns because the deadline was set to `.never`. Deadlines are important so we shouldn't make it easy for people to just ignore them. However if we *do* add defaults again they shouldn't be `.never` they should be sensible defaults for that particular use case.